Makefile

```
@@ -1,65 +1,44 @@
-tidy: ##@helper  ensures that the go.mod file matches the source code in the module
-	go mod tidy -v
-
-.PHONY: package
-package: build ##@application creates packaged versions (zip, tar.gz) from the binary
-	zip -r $(BINARY_NAME).zip $(BINARY_NAME)
-	tar czf $(BINARY_NAME).tar.gz $(BINARY_NAME)
-
-# ignore unknown commands
-%:
-    @:
+.PHONY: test test-unit test-integration test-coverage update-golden clean build smoke
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)